CourseSearchAPIHttpTrigger/search.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import requests
+import os
+import json
+import sys
+import inspect
+
+
+# TODO investigate setting PATH in Azure so can remove this
+CURRENTDIR = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+PARENTDIR = os.path.dirname(CURRENTDIR)
+sys.path.insert(0, CURRENTDIR)
+sys.path.insert(0, PARENTDIR)
+
+import exceptions
+
+
+def find_postcode(url, api_key, api_version, index_name, postcode):
+
+    try:
```

Comment:
Not sure what the try except is doing here?

Reply:
Removing after conversation with @philratcliffe - no need to pass exceptions up the chain; this happens by default